### DISCUSSION TITLE
How do I change the size of text chunks?

### DISCUSSION THREAD

**Discussion by Moorky** [deepdoctection/deepdoctection#226] [Q&A]
I want the layout detector to make bigger chunks out of my text blocks, how do I do that? I already tried adjusting everything in the config.

Here is the issue visualized: [Screenshot](https://ibb.co/64dr2rJ)

As you can see, there are too many text boxes in the second paragraph and I want the layout boxes to capture the whole text as one box.

I appreciate any help!

**Comment by JaMe76** [comment]
Hi,

there is nothing implemented that will give you larger chunks (unless you want the full narrative text).

One idea that will group all boxes with the same label is to write a post processing function that loops over all layout sections sorted by ascending ˋreading_orderˋ and groups all text of consecutive same label. 

However, this might give you some unexpected behavior, as well…
